### PR TITLE
render notebook step actions in the natural order

### PR DESCRIPTION
**Pull request by uladzimirdev**
[Epic](https://github.com/metabase/metabase/issues/47911)

Changes order of the action buttons to the more natural

Screenshot 2024-09-25 at 00 11 36
Screenshot 2024-09-25 at 00 11 47

Screenshot 2024-09-25 at 00 12 02
Screenshot 2024-09-25 at 00 12 23
Screenshot 2024-09-25 at 00 12 48

### CODE REVIEW

File: frontend/src/metabase/querying/notebook/components/NotebookStepList/NotebookStepList.unit.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+import { NotebookStepList } from "./NotebookStepList";
+
+const ORDERS_PRODUCT_JOIN_CONDITION: JoinCondition = [
+  "=",
+  ["field", ORDERS.PRODUCT_ID, null],
+  ["field", PRODUCTS.ID, { "join-alias": "Products" }],
+];
+
+const PRODUCTS_JOIN = {
```
Comment (ranquild) [inline frontend/src/metabase/querying/notebook/components/NotebookStepList/NotebookStepList.unit.spec.tsx:38]: We don't use MBQL when writing unit tests now. Please use MBQL lib for that https://github.com/metabase/metabase/blob/8486a9f9c85b4820c99249468286ead200a79c9c/frontend/src/metabase/querying/filters/hooks/use-filter-modal/utils/filters.unit.spec.ts#L9

**Review by ranquild — COMMENTED** [review]
Please don't use MBQL in tests

File: frontend/src/metabase/querying/notebook/components/NotebookStepList/NotebookStepList.unit.spec.tsx (hunk 1 above)
Comment (uladzimirdev) [reply frontend/src/metabase/querying/notebook/components/NotebookStepList/NotebookStepList.unit.spec.tsx:38]: but I use MBQL to create cards, not queries. do we have a way to create a card using metabase-lib?

Comment (ranquild) [reply frontend/src/metabase/querying/notebook/components/NotebookStepList/NotebookStepList.unit.spec.tsx:38]: `new Question(createMockCard(), metadata).setQuery(...)`

Comment (uladzimirdev) [reply frontend/src/metabase/querying/notebook/components/NotebookStepList/NotebookStepList.unit.spec.tsx:38]:
```
const question = Question.create({ databaseId, metadata });
question = question.setQuery(query);
```

just found it, thanks

**Review by ranquild — APPROVED** [review]

**Pull request merged**